### PR TITLE
Add missing descriptions for /zh/docs/tasks

### PR DESCRIPTION
**Pull request by howieyuen**
**Problem:**

Chinese page /zh/docs/tasks/ is outdated, which are missing description comparing with English version.

**Proposed Solution:**

Add missing description for each subjects in tasks. There are 14 descriptions missed totally. Following table indicates more explicity:

**Page to Update:**

https://kubernetes.io/zh/docs/tasks/

**Related Issue:**

https://github.com/kubernetes/website/issues/23932

### CODE REVIEW

File: content/zh/docs/tasks/access-application-cluster/_index.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
 title: 访问集群中的应用程序
 weight: 60
+description: 配置负载平衡，端口转发或设置防火墙或 DNS 配置，以访问集群中的应用程序。
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/_index.md:4]:
```suggestion
description: 配置负载平衡、端口转发或设置防火墙或 DNS 配置，以访问集群中的应用程序。
```

File: content/zh/docs/tasks/configure-pod-container/_index.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ---
 title: "配置 Pods 和容器"
 weight: 20
+description: 对Pod和容器执行常见的配置任务。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/_index.md:4]: 空格

File: content/zh/docs/tasks/debug-application-cluster/_index.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ---
 title: "监控、日志和排错"
 weight: 80
+description: 设置监视和日志记录以对集群进行故障排除，或调试容器化的应用程序。
```
Comment (tengqm) [inline content/zh/docs/tasks/debug-application-cluster/_index.md:4]:
```suggestion
description: 设置监视和日志记录以对集群进行故障排除或调试容器化应用。
```

File: content/zh/docs/tasks/job/_index.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
 ---
 title: "运行 Jobs"
 weight: 50
+description: 使用并行处理运行 Jobs 。
```
Comment (tengqm) [inline content/zh/docs/tasks/job/_index.md:4]: 多余空格

File: content/zh/docs/tasks/manage-hugepages/scheduling-hugepages.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ reviewers:
 - derekwaynecarr
 title: 管理巨页（HugePages）
 content_type: task
----
+description: 将大页配置和管理为集群中的可调度资源。
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-hugepages/scheduling-hugepages.md:6]: 这里的 `---` 要保留，加到第 6 行之后。

File: content/zh/docs/tasks/manage-kubernetes-objects/_index.md [hunk 6]
```diff
@@ -1,4 +1,5 @@
 ---
 title: "管理 Kubernetes 对象"
 weight: 25
+description: 与声明式和命令式 Kubernetes API 交互的范例。
```
Comment (tengqm) [inline content/zh/docs/tasks/manage-kubernetes-objects/_index.md:4]: 这句英文是什么？

Comment (howieyuen) [reply content/zh/docs/tasks/manage-kubernetes-objects/_index.md:4]: Set up monitoring and logging to troubleshoot a cluster, or debug a containerized application.

Comment (tengqm) [reply content/zh/docs/tasks/manage-kubernetes-objects/_index.md:4]: 用声明式和命令式范型与 Kubernetes API 交互

**Pull request merged**